### ISSUE TITLE
V.06b6: error in log when queue finished

### ISSUE THREAD

**Issue by erkr** [music-assistant/hass-music-assistant#356]
Error logged once a queue is completed:

``` 
Logger: aiohttp.server
Source: /usr/local/lib/python3.9/site-packages/aiohttp/web_protocol.py:405 
First occurred: 16:45:08 (1 occurrences) 
Last logged: 16:45:08

Error handling request
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/aiohttp/web_protocol.py", line 435, in _handle_request
    resp = await request_handler(request)
  File "/usr/local/lib/python3.9/site-packages/aiohttp/web_app.py", line 504, in _handle
    resp = await handler(request)
  File "/usr/local/lib/python3.9/site-packages/music_assistant/controllers/streams.py", line 160, in serve_queue_stream
    await queue_stream.subscribe(client_id, resp.write)
  File "/usr/local/lib/python3.9/site-packages/music_assistant/controllers/streams.py", line 311, in subscribe
    assert not self.done.is_set(), "Stream task is already finished"
AssertionError: Stream task is already finished
```

**Comment by marcelveldt** [comment]
should be fixed in b7 but I did not really test it

**Comment by erkr** [comment]
Finally, I  got to the point of verifying this one. Previous test attempts all ended up in various queuing issues. 
Yes Solved in 2022.6.2 and probably earlier ;-)
